test/controllers/experiment_controller_test.exs

```diff
@@ -17,65 +17,55 @@ defmodule Xee.ExperimentControllerTest do
   end
 
   test "get as a participant without experiment" do
-    x_id = "test"
+    token = "test"
```

Comment:
Please use /experiment/:xid in this case, and add a case with a description like "get as a participant with [using] shortcut token".